CRM/Event/Form/Registration/Register.php

```diff
@@ -888,7 +888,7 @@ public static function formRule($fields, $files, $form) {
         $isZeroAmount = TRUE;
       }
 
-      if ($isZeroAmount && !($form->_forcePayement && !empty($fields['additional_participants']))) {
+      if ($isZeroAmount) {
```

Comment:
hang on have I got this right - $form->_forcePayement was ALWAYS false so it was never the case that the second condition was true & hence it was meaningless

Reply:
I think that is right unless it works becasue !FALSE but urg that is annoying as hell might be worth just testing with a small PHP script of something like
```
if (!(FALSE && !empty(['hi'])) {
  print_r('See hi');
}
else {
  print_r('So it wasn't validating');
}
```

Reply:
doing testing found that even if the 2nd part of that inner IF was TRUE the whole section was returning TRUE because forcePayment was false so Not FALSE is TRUE so makes sense to clear it all out.

Reply:
it was a mind mess though!